### ISSUE TITLE
md-button with md-icon in md-table

### ISSUE THREAD

**Issue by Leshgan** [vuematerial/vue-material#534]
When using md-button with md-icon inside it's ok.
But when the same button inside table, its losing style.
Example here http://jsbin.com/hejulu/edit?html,output

**Comment by Birssan** [comment]
Yep same here, i've been meaning to make a PR for this.
You need to override the margin on the icon element like this.

```
.md-table .md-table-cell .md-button .md-icon {
    margin: auto;
}
```

**Comment by Birssan** [comment]
Noticed the same issue when an icon button is used as a menu trigger. The problem doesn't occur in the doc as there is added CSS.

**Comment by pablohpsilva** [comment]
Please follow the table fixes on PR #783 :)
